### PR TITLE
Connect announcements api

### PR DESCRIPTION
**Pull request by j-palos**

### CODE REVIEW

File: backend/routes/announcements.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ const router = express.Router();
 const { Announcements} = require('../database');
 
 // Routes
+
+router.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
```
Comment (rquinones93) [inline backend/routes/announcements.js:11]: Please refactor line 7 to the following to stick with convention of the other route files
`router.use( ( request, response, next ) => {`

Rename `res` on lines 8 and 9 to `response`

**Review by rquinones93 — CHANGES_REQUESTED** [review]
Unsure on the React files, but just a small change on the announcements route

File: backend/routes/announcements.js (hunk 1 above)
Comment (j-palos) [reply backend/routes/announcements.js:11]: Done.

**Review by rquinones93 — APPROVED** [review]

**Comment by rquinones93** [comment]
I'm assuming styling for announcements is work for another student. As of now everything looks good to me. 👍 

Going to look up how to run both the React and Node apps at the same time, make it easier on people.

**Comment by j-palos** [comment]
> I'm assuming styling for announcements is work for another student. As of now everything looks good to me. 👍
> 
> Going to look up how to run both the React and Node apps at the same time, make it easier on people.

Yeah, I imagine there's a way to script both of them together. I can look into it as well.

**Pull request merged**